trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf5340/partition/flash_layout.h

```diff
@@ -30,7 +30,8 @@
  * 0x000f_0000 Protected Storage Area (16 KB)
  * 0x000f_4000 Internal Trusted Storage Area (8 KB)
  * 0x000f_6000 NV counters area (4 KB)
- * 0x000f_7000 Unused
+ * 0x000f_7000 Unused (4 KB)
+ * 0x000f_8000 Non-secure storage (32 KB)
```

Comment:
Could this storage be secure? The commit message states that "If newly added CMake variable NS_STORAGE=ON then configure  the storage area to be non-secure." To me that implies that this area could also be secure storage.

Reply:
Ok, I can restate it to "then add a non-secure region for the storage"